packages/cli/src/commands/certs/add.ts

```diff
@@ -0,0 +1,119 @@
+import color from '@heroku-cli/color'
+import {APIClient, Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import {meta} from '../../lib/certs/endpoints'
+import {waitForDomains} from '../../lib/certs/domains'
+import {prompt} from 'inquirer'
+import {getCertAndKey} from '../../lib/certs/get_cert_and_key'
+import {cert, getCertificateDetails} from '../../lib/certs/certificate_details'
+
+async function getMeta(app: string) {
+  return meta(app, 'sni', undefined)
+}
+
+async function configureDomains(app: string, heroku: APIClient, cert: Heroku.SniEndpoint) {
+  const certDomains = cert.ssl_cert.cert_domains
+  const apiDomains = await waitForDomains(app, heroku)
+  const appDomains = apiDomains?.map((domain: Heroku.Domain) => domain.hostname as string)
+  const matchedDomains = matchDomains(certDomains, appDomains ?? [])
+  if (matchedDomains.length > 0) {
+    ux.styledHeader('Almost done! Which of these domains on this application would you like this certificate associated with?')
+    const selections = await prompt<{domains: string[]}>([{
+      type: 'checkbox',
+      name: 'domains',
+      message: 'Select domains',
+      choices: matchedDomains,
+    }])
+    await Promise.all(selections?.domains.map(domain => {
+      return heroku.patch(`/apps/${app}/domains/${domain}`, {
+        body: {sni_endpoint: cert.name},
+      })
+    }))
+  }
+}
+
+export default class Add extends Command {
+  static topic = 'certs'
+  static strict = true
+  static description = 'add an SSL certificate to an app'
+  static help = 'Note: certificates with PEM encoding are also valid'
+  static flags = {
+    bypass: flags.boolean({
+      description: 'bypass the trust chain completion step',
+      deprecated: {
+        message: 'use of the --bypass flag is deprecated. The flag currently does not perform any additional behavior. Please remove --bypass',
+      },
+    }),
```

Comment:
Since this deprecation message has been part of the functionality here for a long time, could you please just remove this flag?